static/css/shop/_filters-spec.scss

```diff
@@ -0,0 +1,139 @@
+.price {
+  flex-direction: column;
+  align-items: stretch;
+}
+
+.price > div {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.price > .btn {
+  align-self: flex-end;
+}
+
+.price input {
+  width: 4em;
+}
+
+.color-grid {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-end;
+}
+
+.color-grid > label {
+  margin: 5px;
+}
+
+#chk-000000 > input + .icon {
+  background: #000;
+  border-color: #000;
+  --r: 0;
+  --g: 0;
+  --b: 0;
+}
+
+#chk-a7a7a7 > input + .icon {
```

Comment:
That's hardly maintainable. Those IDs were meant for demo, they shouldn't end up in the actual frontend